js/index.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
 		"eslint:recommended",
 		"plugin:import/errors",
 		"plugin:import/warnings",
-		...rules
+		...rules,
+		"prettier",
```

Comment:
@ascartabelli The docs recommend this order, but I feel that we should put our rules last, such that we have the final word.  It might increase the maintenance needed on our side, though.  What do you think?

Reply:
> @ascartabelli The docs recommend this order, but I feel that we should put our rules last, such that we have the final word. It might increase the maintenance needed on our side, though. What do you think?

I've always kept my rules as the ones having the last word.
Is there any specific reason stated in their recommendation?